linux/vcpkg_config_manager/vcpkg.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "boinc-client",
+    "version-string": "7.19.0",
```

Comment:
Since you added a new vcpkg.json file, next script should be updated: https://github.com/BOINC/boinc/blob/master/set-version.py

Reply:
ok.

Reply:
Waiting for https://github.com/BOINC/boinc/pull/4758

Reply:
done.

Reply:
done.